src/main.ts

```diff
@@ -7,20 +7,80 @@ dotenv.config()
 const octokit = new Octokit({ auth: process.env.ACCESS_TOKEN })
 
 const titles = [
+  'docs(ja): guides/components-glossary/pages-middleware.md',
+  'docs(ja): guides/components-glossary/pages-scrolltotop.md',
+  'docs(ja): guides/components-glossary/pages-transition.md',
+  'docs(ja): guides/components-glossary/pages-validate.md',
+  'docs(ja): guides/components-glossary/pages-watchquery.md',
+  'docs(ja): guides/concepts/nuxt-lifecycle.md',
+  'docs(ja): guides/concepts/server-side-rendering.md',
+  'docs(ja): guides/concepts/static-site-generation.md',
   'docs(ja): guides/concepts/views.md',
```

Comment:
,`context-helpers.md` の追加をお願いします

```suggestion
  'docs(ja): guides/concepts/static-site-generation.md',
  'docs(ja): guides/concepts/views.md',
  'docs(ja): guides/concepts/context-helpers.md',
```